webapp/components/test-search.js

```diff
@@ -9,8 +9,28 @@ import { html } from '../node_modules/@polymer/polymer/lib/utils/html-tag.js';
 import { PolymerElement } from '../node_modules/@polymer/polymer/polymer-element.js';
 import { WPTFlags } from './wpt-flags.js';
 import './ohm.js';
+import { DefaultBrowserNames } from './product-info.js';
 
 /* eslint-enable */
+const statuses = [
+  'pass',
+  'ok',
+  'error',
+  'timeout',
+  'notrun',
+  'fail',
+  'crash',
+  'skip',
+  'assert',
+  'unknown',
+];
+
+const atoms = {
+  status: statuses,
+};
+for (const b of DefaultBrowserNames) {
```

Comment:
Another blank line here?
```suggestion

for (const b of DefaultBrowserNames) {
```